infra/src/main/kotlin/fi/fta/geoviite/infra/geocoding/GeocodingDao.kt

```diff
@@ -24,7 +27,11 @@ data class GeocodingContextCacheKey (
         kmPostVersions.forEachIndexed { index, version ->
             kmPostVersions.getOrNull(index+1)?.let { next ->
                 require(next.id.intValue > version.id.intValue) {
-                    "Cache key km-posts must be in order: index=$index version=$version next=$next"
+                    "Cache key km-posts must be in order: " +
```

Comment:
Sanity check, koska näin pitäisi aina olla. Järjestyksellä ei sinänsä ole väliä, mutta sen tulee olla cache-avaimessa aina sama jotta avaimen hash on aina sama.